pythonbible/regular_expressions.py

```diff
@@ -107,24 +108,24 @@ def _add_suffix(regex: str, suffix: str = None) -> str:
     Book.LAMENTATIONS: build_book_regular_expression(
         r"Lam\.*(?:entations)?", suffix=r"of\s+Jeremiah"
     ),
-    Book.EZEKIEL: r"Ezek\.*(?:iel)?",
+    Book.EZEKIEL: r"Eze\.*(?:k\.*(?:iel)?)?",
     Book.DANIEL: r"Dan\.*(?:iel)?",
     Book.HOSEA: r"Hos\.*(?:ea)?",
-    Book.JOEL: "Joel",
-    Book.AMOS: "Amos",
-    Book.OBADIAH: r"Obad\.*(?:iah)?",
+    Book.JOEL: "Joe\.*(?:l)?",
+    Book.AMOS: "Amo\.*(?:s)?",
+    Book.OBADIAH: r"Oba\.*(?:d\.*(?:iah)?)?",
     Book.JONAH: r"Jon\.*(?:ah)?",
     Book.MICAH: r"Mic\.*(?:ah)?",
     Book.NAHUM: r"Nah\.*(?:um)?",
     Book.HABAKKUK: r"Hab\.*(?:akkuk)?",
-    Book.ZEPHANIAH: r"Zeph\.*(?:aniah)?",
+    Book.ZEPHANIAH: r"Zep\.*(?:h\.*(?:aniah)?)?",
     Book.HAGGAI: r"Hag\.*(?:gai)?",
-    Book.ZECHARIAH: r"Zech\.*(?:ariah)?",
+    Book.ZECHARIAH: r"Zec\.*(?:h\.*(?:ariah)?)?",
     Book.MALACHI: r"Mal\.*(?:achi)?",
-    Book.MATTHEW: r"Matt\.*(?:hew)?",
-    Book.MARK: "Mark",
-    Book.LUKE: "Luke",
-    Book.JOHN: r"(?<!(?:1|2|3|I)\s)(?<!(?:1|2|3|I))John",
+    Book.MATTHEW: r"Mat\.*(?:t\.*(?:hew)?)?",
+    Book.MARK: r"Mar\.*(?:k)?",
+    Book.LUKE: r"Luk\.*(?:e)?",
+    Book.JOHN: r"(?<!(?:1|2|3|I)\s)(?<!(?:1|2|3|I))"+JOHN_REGULAR_EXPRESSION
```

Comment:
Looks like this line is missing the comma at the end.

Reply:
Should be fixed, added `Act\.*(?:s)?` as well.